### DISCUSSION TITLE
Feature request:  Ability to hide the bottom pane (containing results/execution log etc)

### DISCUSSION THREAD

**Discussion by jonnyry** [dbeaver/dbeaver#198] [Q&A] [answered]
When writing long SQL scripts it is useful to have the full height of the application to see you script.  It would be useful if it was possible to hide/close the bottom pane containing the Results/Explain/Execution Log/Output tabs temporarily.

Even better if you could assign a key shortcut to toggle the visibility of this pane.

**Comment by serge-rider** [comment, marked as answer]
Its already there.
"Toggle results panel" in the context menu or F6.

**Comment by jonnyry** [comment]
Excellent, thanks!

**Comment by karanparashar1** [comment]
It is not working in my DBeaver Ultimate :((
<img width="960" alt="image" src="https://github.com/dbeaver/dbeaver/assets/65452093/33104d02-e5b8-4e23-98db-9bc03119ffb2">
